docs/source/advanced_features/configuration.rst

```diff
@@ -0,0 +1,46 @@
+#######################
+ Request Configuration
+#######################
+
+TensorBay provides lots of convenient services. The default setting of TensorBay service should be
+sufficient for most use cases. This topic mainly describes the available
+:class:`~tensorbay.client.requests.Config` options for customizing request, they are:
+
+.. _request_config_table:
+
+.. table:: Requests Configuration Tables
+   :align: center
+   :widths: auto
+
+   ===================== ============================================================
+   Variables             Description                                              
+   ===================== ============================================================
+   max_retries           | The number of maximum retry times of the request.
+                         | If the request method is one of the allowed_retry_methods,
+                         | the response status is in one of the allowed_retry_status,
+                         | then the request can auto-retry `max_reties` times.
+                         | Scenario: poor network connection.
+                         | Default: 3 times.
+   allowed_retry_methods | The allowed methods for retrying request.
+                         | Default: ["HEAD", "OPTIONS", "POST", "PUT"]
+   allowed_retry_status  | The allowed status for retrying request.
+                         | Default: [429, 500, 502, 503, 504]
+   timeout               | The number of seconds before the request times out.
+                         | Scenario: poor network connection.
+                         | Default: 30 seconds.
+   is_internal           | Whether the request is from internal or not.
+                         | Scenario: faster speed.
```

Comment:
Set it to ``True`` when dataset and model are in the same OSS endpoint. See :ref: ... for details.